node-src/errorMonitoring.ts

```diff
@@ -1,15 +1,62 @@
 import * as Sentry from '@sentry/node';
+import stripAnsi from 'strip-ansi';
+
+/**
+ * Remove ANSI escape codes from Sentry ErrorEvents.
+ *
+ * @param event An event containing a captured exception.
```

Comment:
```suggestion
 * @param event A {@link Sentry.ErrorEvent} containing a captured exception.
```

Reply:
Should we add those to the breadcrumb function too?